infrastructure/stacks/uec-developer-role/uec-rw-data-services.json

```diff
@@ -28,6 +28,7 @@
                 "redshift:DescribeEvents",
                 "redshift-serverless:ListWorkgroups",
                 "redshift:CreateSavedQuery",
+                "backup:*",
```

Comment:
Might be my misunderstanding, but I was expecting to see these permissions:
"dynamodb:CreateBackup",
"dynamodb:RestoreTableFromBackup",

However it looks like those should already be covered by the 'VisualEditor4' policy below (line 117)

Reply:
Possible duplicate reply. Those permissions are covered by dynamodb.* but by themselves they were not sufficient - hence the blanket inclusion of backup